modules/ROOT/pages/getting-started.adoc

```diff
@@ -7,63 +7,99 @@ xref:glossary:index.adoc[glossary].
+build PipelineRuns that are defined in your git repositories, view and iterate
+on the results of your IntegrationTestScenarios, and create Releases to release
+specific Snapshots according to specific ReleasePlans.
+
+NOTE: You can grant, modify, or remove other users' access to any namespace for
+which you have the `admin` role. Each namespace's admins have the
+responsibility to ensure that appropriate access is granted to users. If users
+are over-privileged, they might have access to any added secrets or to perform
+other unintended actions.
 
 ==== Managed namespace
-Sometimes users will need to perform actions that require credentials in someone else's control. Since admins
-in a tenant namespace can see all secrets, these actions need to be performed via separate managed namespaces.
-The primary interaction mode between tenant and managed namespaces is to create a Release referencing a specific
-Snapshot which will trigger a specific pipeline in the managed namespace.
+These are namespaces where a privileged release engineering team or
+site-reliability engineering team manages release pipelines and credentials for
+your organization.
+
+The primary interaction mode between tenant and managed namespaces is to create
+a Release in a tenant namespace referencing a specific Snapshot which will
+trigger a specific release pipeline in the managed namespace.
+
+=== Custom Resource (CR)
+In Kubernetes, a
+link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom
+Resources (CR)] is an extension of the Kubernetes API.
+
+All {ProductName} APIs are implemented as Kubernetes CRs. If
+you're familiar with common Kubernetes resources such as Pods and Deployments,
+you'll find that {ProductName} resources appear on cluster in the same way:
+Applications, Components, Snapshots, and PipelineRuns.
+
+A consequence of this fact is that commonplace Kubernetes client tools such as
+link:https://kubernetes.io/docs/reference/kubectl/[kubectl] understand and can
+work with {ProductName} APIs.
 
 === Component
 A Component CR describes the properties for an OCI artifact including the git repository where the artifact is from,
-its latest built commit, initial build configuration parameters (i.e. Containerfile path), and any relationships to
+its latest built commit, initial build configuration parameters, and any relationships to
 other Components. The CR also contains a reference to its single owning Application.
-Component names must be unique in a namespace, even when components are used in different application.
-
-=== Pipelines as Code (PAC)
-https://pipelinesascode.com/[Pipelines as Code] is a project that provides opinionated tooling to define a
-Tekton-based CI/CD pipeline. It enables the use of a Tekton PipelineDefinition within a git repository and for
-that pipeline to be triggered upon new commits/PR to the repository. In Konflux, these pipelines are pushed to
-the git repositories in the `.tekton` directory.
```

Comment:
Is there motivation for removing this? Is there a better place for it to be moved to?

Reply:
Yeah, it just didn't seem necessary to describe PaC here. It felt like an odd duck in the list.

I moved mention of the .tekton directory to the build pipeline section.